rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/rascal/RascalLanguageServer.java

```diff
@@ -29,13 +29,21 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.rascalmpl.vscode.lsp.BaseLanguageServer;
 
 public class RascalLanguageServer extends BaseLanguageServer {
     public static void main(String[] args) {
-        startLanguageServer(() -> {
-            ExecutorService threadPool = Executors.newCachedThreadPool();
-            return new RascalTextDocumentService(threadPool);
-        }, 8888);
+        try {
+            startLanguageServer(() -> {
+                ExecutorService threadPool = Executors.newCachedThreadPool();
+                return new RascalTextDocumentService(threadPool);
+            }, 8888);
+        }
+        catch (Throwable e) {
+            final Logger logger = LogManager.getLogger(RascalLanguageServer.class);
+            logger.debug(e.getMessage(), e);
```

Comment:
I think this should be of level `fail` or `error`? not debug? Otherwise we might miss this in deployment setups.

Reply:
right! that's a good catch